src/cookieSync.js

```diff
@@ -18,9 +20,33 @@ function triggerPixel(url) {
   img.src = url;
 }
 
+function insertUserSyncIframe(url) {
```

Comment:
a similar function is available in util, use the one from util.
mention height and width as zero
some browsers have default CSS applied to elements which can make a 0x0 iframe visible due to attributes like a border, make sure that this iframe is invisible

Reply:
Done